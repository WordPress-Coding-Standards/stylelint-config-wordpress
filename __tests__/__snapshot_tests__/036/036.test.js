'use strict';

const stylelint = require( 'stylelint' ),
	systemTestUtils = require( '../systemTestUtils.js' );

it( '036', () => {
	return stylelint.lint({
		files: [systemTestUtils.caseStylesheetGlob( '036' )],
		configFile: systemTestUtils.caseConfig( '036' ),
	}).then( ({ results }) => {
		expect( systemTestUtils.prepResults( results ) ).toMatchSnapshot();
	});
});
