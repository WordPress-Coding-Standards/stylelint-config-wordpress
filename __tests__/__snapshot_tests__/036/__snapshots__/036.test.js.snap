// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`036 1`] = `
Array [
  Object {
    "deprecations": Array [],
    "errored": undefined,
    "ignored": undefined,
    "invalidOptionWarnings": Array [],
    "parseErrors": Array [],
    "source": "036/stylesheet.css",
    "warnings": Array [],
  },
]
`;
